src/data_traits.rs

```diff
@@ -96,6 +96,15 @@ pub unsafe trait Data: RawData {
     where
         Self::Elem: Clone,
         D: Dimension;
+
+    fn to_share<D>(self_: &ArrayBase<Self, D>) -> ArrayBase<OwnedArcRepr<Self::Elem>, D>
```

Comment:
```suggestion
    #[doc(hidden)]
    fn to_share<D>(self_: &ArrayBase<Self, D>) -> ArrayBase<OwnedArcRepr<Self::Elem>, D>
```

Like the rest of the methods. And it should have a doc comment here still - as documentation.

Reply:
Yes, the corresponding modifications have been added.